### ISSUE TITLE
documentation issue

### ISSUE THREAD

**Issue by ScotsScripts** [Birssan/grav-plugin-about-me#8]
If you add {% include 'partials/aboutme.html.twig' %} to a page, all you see on the page when viewed is  {% include 'partials/aboutme.html.twig' %}

It took me two hours to figure out that I had to turn on TWIG processing in the page config to get that statement above to work.

For experienced GRAV users this is probably obvious, but this is my first time using GRAV and I didn't know you had to turn on TWIG processing for the plugin to work. Please update the documentation with beginners in mind, thanks!

**Comment by seb-vial** [comment]
Hello there,

This is not really an issue, what you did is applicable to every plugin you'll use that uses templates.
Basically this plugin is designed to be used in a template of your theme, of course you can use it in a single page as you did, but activating the TWIG functionality doesn't depend on this plugin or any other for that matter.

Activating TWIG in pages is a Grav functionality, you can find some explanation in the official documentation, specifically how yo activate TWIG in a page https://learn.getgrav.org/admin-panel/page-advanced#overrides

![twig](https://cloud.githubusercontent.com/assets/5689193/24343671/f63c86ec-12c7-11e7-9b07-496fdad0798d.PNG)

![image](https://cloud.githubusercontent.com/assets/5689193/24343679/037f4006-12c8-11e7-85e7-ec6712e114f4.png)

To sum up there is no need to add this in the documentation for this plugin as it's not specific to this plugin.
